provisioning_client/deps/CMakeLists.txt

```
@@ -57,6 +76,9 @@ else()
             add_definitions(-DGB_MEASURE_MEMORY_FOR_THIS -DGB_DEBUG_ALLOC)
             add_definitions(-DWIN32)
         ENDIF(WIN32)
+		
+		#Set RIoT ECDH Curve
+		add_definitions(-DRIOTSECP256R1)
```

This should probably be `target_compile_definitions`
```suggestion
		target_compile_definitions(msr_riot RIOTSECP256R1)
```

---

Fixed